tests/monster_test.schema.json

```diff
@@ -0,0 +1,120 @@
+{
+"$schema": "http://json-schema.org/draft-04/schema#",
+"definitions": {
+  "MyGame_OtherNameSpace_FromInclude" : {
+    "type" : "string",
+    "enum": ["IncludeVal"]
+  },
+  "MyGame_Example_Color" : {
+    "type" : "string",
+    "enum": ["Red", "Green", "Blue"]
+  },
+  "MyGame_Example_Any" : {
+    "type" : "string",
+    "enum": ["NONE", "Monster", "TestSimpleTableWithEnum", "MyGame_Example2_Monster"]
+  },
+
+"MyGame_OtherNameSpace_Unused" : {
+  "type" : "object",
+  "description" : "",
+  "properties" : {
+    }
+  },
+
+"MyGame_Example2_Monster" : {
+  "type" : "object",
+  "description" : "",
+  "properties" : {
+    }
+  },
+
+"MyGame_Example_Test" : {
+  "type" : "object",
+  "description" : "",
```

Comment:
if description is empty, should it maybe be left out to reduce clutter? Same of any other fields that are optional?